descartes_light/solvers/include/descartes_light/solvers/bgl/impl/dfs_add_all_solver.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2021, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef DESCARTES_LIGHT_SOLVERS_BGL_IMPL_ADD_ALL_DYNAMIC_DIJKSTRA_SOLVER_HPP
+#define DESCARTES_LIGHT_SOLVERS_BGL_IMPL_ADD_ALL_DYNAMIC_DIJKSTRA_SOLVER_HPP
+
+#include <descartes_light/descartes_macros.h>
+DESCARTES_IGNORE_WARNINGS_PUSH
+#include <console_bridge/console.h>
+#include <omp.h>
+#include <boost/graph/depth_first_search.hpp>
+#include <descartes_light/solvers/bgl/dfs_add_all_solver.h>
+#include <descartes_light/solvers/bgl/impl/event_visitors.hpp>
+#include <descartes_light/types.h>
```

Comment:
Need to add call to `DESCARTES_INGORE_WARNINGS_POP` at the bottom of this snippet